scripts/import_srpm.py

```diff
@@ -1,20 +1,34 @@
 #!/usr/bin/env python3
 import argparse
+import logging
 import os
 import subprocess
 
+def call_process(args):
```

Comment:
`call` is probably clear enough in that context, and pleasantly shorter

Reply:
Hm, I'm a bit wary of using generic names, especially when it's so cheap to be more specific